packages/react-components/react-avatar/src/components/AvatarGroupItem/useAvatarGroupItemStyles.ts

```diff
@@ -1,42 +1,99 @@
+import { useSizeStyles } from '../../Avatar';
+import { AvatarGroupContext } from '../../contexts/AvatarGroupContext';
 import { makeStyles, mergeClasses, shorthands } from '@griffel/react';
-import { tokens } from '@fluentui/react-theme';
+import { tokens, typographyStyles } from '@fluentui/react-theme';
+import { useContextSelector } from '@fluentui/react-context-selector';
+import type { AvatarSizes } from '../../Avatar';
 import type { AvatarGroupProps } from '../../AvatarGroup';
 import type { AvatarGroupItemSlots, AvatarGroupItemState } from './AvatarGroupItem.types';
-import type { AvatarSizes } from '../../Avatar';
 import type { SlotClassNames } from '@fluentui/react-utilities';
-import { useContextSelector } from '@fluentui/react-context-selector';
-import { AvatarGroupContext } from '../../contexts/AvatarGroupContext';
 
 export const avatarGroupItemClassNames: SlotClassNames<AvatarGroupItemSlots> = {
   root: 'fui-AvatarGroupItem',
   avatar: 'fui-AvatarGroupItem__avatar',
   overflowLabel: 'fui-AvatarGroupItem__overflowLabel',
 };
 
+const avatarGroupItemDividerWidthVar = '--fuiAvatarGroupItem--divierWidth';
```

Comment:
I don't think we have explicit naming conventions for CSS vars but I think we should stick with precedent and use the [convention from Slider](https://github.com/microsoft/fluentui/blob/master/packages/react-components/react-slider/src/components/Slider/useSliderStyles.ts#L15-L25).

That would make the name of this var `--fui-AvatarGroupItem__divider--width`